strawberry/schema/schema_converter.py

```diff
@@ -64,9 +68,11 @@ def serialize(self, output_value: Any) -> str:
 class GraphQLCoreConverter:
     # TODO: Make abstract
 
-    def __init__(self, config: StrawberryConfig):
+    def __init__(self, schema: "Schema"):
         self.type_map: Dict[str, ConcreteType] = {}
-        self.config = config
+        self.config = schema.config
+        self.schema = schema
```

Comment:
do you think we can find a way to decouple the converter from the schema?
now we have a circular dependency between schema and schema converter, not sure this is ideal /cc @BryceBeagle

Reply:
Yeah I'm not sure if instantiating the converter _with_ the schema is a good idea. imo the converter should be a separate entity. I do see the problem that as implemented, `from_scalar` needs to call `Schema.get_scalar`. :thinking:

Reply:
Yeah don't know how to do this any other way. We need ways to define hooks on the GraphQL convertor so that we can change functionality. We could pass the "get_scalar" hook as an extension instead of subclassing the schema?

Reply:
I've changed the approach here so that we don't have this dependency. Now you pass a map of scalar overrides to the schema which then gets passed to the schema convertor. I think this is a better approach.